projects/HMR/src/cl_HMR.cpp

```diff
@@ -2405,6 +2405,12 @@ namespace moris::hmr
             const uint               aMesh_Index,
             const uint               aBsplineMeshIndex )
     {
+#if 0 // GAH
+GAH : 
+This code brings in moris::mtk::Mapper
```

Comment:
If we do make an interface to the mapper, it might make sense to put the current implementation in HMR since we hard-code it to use HMR meshes anyways. But this means creating another separate HMR mapper directory to compile that does use FEM so it's not always brought in here. I don't think this is bad and would probably work, but maybe @kkmaute has a better idea.